application/analyst/services/chapter_indexing_service.py

```
@@ -106,22 +97,19 @@ async def index_chapter_summary(
         # 构造 payload
         payload = {
             "chapter_number": chapter_number,
             "text": summary,
             "kind": "chapter_summary",
             "novel_id": novel_id
         }
-
-        point_id = f"{novel_id}_ch{chapter_number}_summary"
-
-        # 写入向量存储
+        # 构造唯一 ID（Qdrant 要求 UUID 或 uint64，用 uuid5 生成确定性 UUID）
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)